tools/device-listing/DeviceInfo.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Threading;
+
+namespace Iot.Tools.DeviceListing
+{
+    class DeviceInfo : IComparable<DeviceInfo>
+    {
+        public string Title { get; private set; }
+        public string ReadmePath { get; private set; }
+
+        public DeviceInfo(string readmePath)
+        {
+            ReadmePath = readmePath;
+            Title = GetTitle(readmePath);
+        }
+
+        public int CompareTo(DeviceInfo other)
+        {
+            return Title.CompareTo(other.Title);
+        }
+
+        private static string GetTitle(string readmePath)
+        {
+            string[] lines = File.ReadAllLines(readmePath);
+            if (lines[0].StartsWith("# "))
```

Comment:
I guess it's fine since we control this, but it might be better here if we just change this to have the `#` be replaced with a ` ` and then we trim the beginning spaces, that way we will work even when more than one space is left.

Reply:
@joperezr I prefer it this way since you will get warnings when README is misformatted - I believe we should fix misformatting instead of ignoring it here